src/main/kotlin/no/nav/familie/ef/sak/journalføring/JournalføringService.kt

```diff
@@ -163,6 +158,25 @@ class JournalføringService(
         return opprettSaksbehandlingsoppgave(behandling, saksbehandler)
     }
 
+    private fun validerJournalføringNyBehandling(
+        journalpost: Journalpost,
+        journalføringRequest: JournalføringRequest
+    ) {
+        feilHvis(journalpost.harStrukturertSøknad() && journalføringRequest.behandling.årsak != null) {
+            "Kan ikke sende inn årsak når journalposten har strukturert søknad"
+        }
+        brukerfeilHvis(!journalpost.harStrukturertSøknad() && journalføringRequest.behandling.årsak == null) {
+            "Må sende inn behandlingsårsak når journalposten mangler digital søknad"
+        }
+        brukerfeilHvis(
+            !journalpost.harStrukturertSøknad() &&
+                journalføringRequest.behandling.årsak == BehandlingÅrsak.PAPIRSØKNAD &&
+                journalpost.avsenderMottaker == null
```

Comment:
Tror ikke du trenger de to første sjekkene. Hvis avsendermottaker er "null" så vil vi aldri få ferdigstilt journalposten uansett.

Reply:
(Men det er 99% sannsynlig at dette hovedsaklig inntreffer på papirsøknader og ettersendinger i posten)

Reply:
Sant, takk 👍 fikset i [9c1105b](https://github.com/navikt/familie-ef-sak/pull/1720/commits/9c1105b1be13de706253763afbc600ee18c11804)